ILTransform/TestProject.cs

```diff
@@ -435,11 +445,14 @@ public bool HasSameContentAs(TestProject project2)
             }
             for (int refIndex = 0; refIndex < ProjectReferences.Length; refIndex++)
             {
-                string ref1 = ProjectReferences[refIndex];
-                string ref2 = project2.ProjectReferences[refIndex];
+                string ref1 = ReplaceOrUpdateStringWithVars(ProjectReferences[refIndex]);
+                string ref2 = ReplaceOrUpdateStringWithVars(project2.ProjectReferences[refIndex]);
```

Comment:
I don't understand what these replacements are doing.  Won't the below `Contains` have the same value before and after this replacement?